start.R

```diff
@@ -454,14 +462,14 @@ if (any(c("--reprepare", "--restart") %in% argv)) {
       cfg$force_replace    <- TRUE
       if (testOneRegi_region != "") cfg$gms$c_testOneRegi_region <- testOneRegi_region
     }
-
+    if ("--quickRun" %in% argv) system("sed -i 's/rtredg = 1.d-9;/rtredg = 1.d-4;/' config/conopt3.op9")
```

Comment:
How do you make sure this setting is reverted after the run has finished

Reply:
Not sure this works with this line, though:  https://github.com/remindmodel/remind/blob/develop/scripts/start/prepare_and_run.R#L488
The rest seems fine

Reply:
I added a new conopt3 options file to avoid these issues